packages/back-end/src/types/AccessTokenRequest.ts

```diff
@@ -0,0 +1,9 @@
+import { Request } from "express";
+import { OrganizationInterface } from "../../types/organization";
+import { AuditInterface } from "../../types/audit";
+
+// eslint-disable-next-line
+export type AccessTokenRequest<B = any, P = any, Q = any> = Request<P, null, B, Q> & {
```

Comment:
Can we name the generic params so that it is more readable? Maybe something like this:

```ts
export type AccessTokenRequest<ReqBody = any, Params = any, ReqQuery = any> = Request<Params, null, ReqBody, ReqQuery> & {
```

This is a bit confusing because they are in a different order than Express's `Request`. Similar to how Express does it: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express/index.d.ts#L112-L118